content/_data/changelogs/lts.yml

```
@@ -3117,6 +3117,23 @@
     message: |-
       Fix a <em>KeyStores with multiple certificates are not supported</em> error from Jetty when passing certain kinds of certificates, such as domain wildcards (regression in 2.204.3).
 
+- version: 2.204.5
+  date: 2020-03-07
+  changes:
+    - type: major bug
+      references:
+        - url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#53
+          title: full changelog
+        - title: Jetty 9.4.18 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.18.v20190429
+        - title: Jetty 9.4.17 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.17.v20190418
+        - title: Jetty 9.4.16 changelog
+          url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.16.v20190411
+      message: |-
+        Revert Winstone-Jetty from 5.8 to 5.3 to resolve regressions in later Winstone versions.
+        Accepts that the <a href="https://issues.jenkins-ci.org/browse/JENKINS-57888">regression in system logging customization</a> is more acceptable than the regressions from later Winstone versions.
```

```suggestion
       High default maximum form size limit and reverse proxy redirection are restored (regressions in 2.204.3).
       System logging customization defect (<a href="https://issues.jenkins-ci.org/browse/JENKINS-57888">JENKINS-57888 - system logging customization</a>) from Jenkins 2.177...2.203.3 is reintroduced in this version as it is considered as less serious than the fixed regressions.
       It will be fixed again in 2.222.1.
```

Maybe it worth splitting the system logging regression as a separate entry.

---

Thanks.  Done in 16e95e3e3e240051a8b364deef97e036a5206b75